docs/diagrams/InnerWorkings.puml

```diff
@@ -0,0 +1,59 @@
+@startuml
```

Comment:
This png file doesn't seem to have been generated, not too sure why, can you try editing this a bit and pushing again?